ui/app/src/components/DashletCard/DashletCard.tsx

```diff
@@ -68,62 +94,118 @@ export function DashletCard(props: DashletCardProps) {
-      </CardContent>
-      {/* endregion */}
-      {/* region Actions */}
-      {footer && (
-        <CardActions
-          sx={{
-            borderTop: '1px solid',
-            borderColor: 'divider',
-            paddingTop: '4px',
-            paddingBottom: '4px',
-            ...sxs?.footer
+            flexDirection: 'column',
+            flexGrow: 1,
+            overflow: 'auto',
+            height: parseDashletContentHeight(contentHeight),
+            pt: 0,
+            ...sxs?.content
           }}
+          {...cardContentProps}
         >
-          {footer}
-        </CardActions>
-      )}
-      {/* endregion */}
+          {children}
+        </CardContent>
+        {/* endregion */}
+        {/* region Actions */}
+        {footer && (
+          <CardActions
+            sx={{
+              borderTop: '1px solid',
+              borderColor: 'divider',
+              paddingTop: '4px',
+              paddingBottom: '4px',
+              ...sxs?.footer
+            }}
+          >
+            {footer}
+          </CardActions>
+        )}
+        {/* endregion */}
+      </>
+    );
+  };
+
+  return (
+    <Card
+      sx={{
+        display: 'flex',
+        flexDirection: 'column',
+        borderLeft: 5,
+        borderLeftColor,
+        ...sxs?.card,
+        ...(isMaximized && {
+          position: 'absolute',
+          zIndex: 5,
+          top: 0,
+          bottom: 0,
+          left: 0,
+          right: 0
+        })
+      }}
+    >
+      <Accordion expanded={isExpanded} sx={{ marginBottom: '0 !important', boxShadow: 'none' }}>
+        {/* region Header */}
+        {renderCardHeader && (
+          <AccordionSummary
+            expandIcon={
+              isMaximized || !collapsible ? null : (
+                <IconButton onClick={() => setIsExpanded(!isExpanded)}>
+                  <ExpandMoreIcon />
+                </IconButton>
+              )
+            }
+            sx={{
+              pl: 0,
+              background: 'none !important',
+              cursor: 'default !important',
+              '& .MuiAccordionSummary-content': { m: 0 }
+            }}
+          >
+            <CardHeader
+              title={title}
+              titleTypographyProps={{ variant: 'h6', component: 'h2' }}
+              action={
+                <>
+                  {maximizable && (
+                    <IconButton onClick={() => updateMaximized()}>
+                      {isMaximized ? <CloseFullscreenOutlinedIcon /> : <OpenInFullOutlinedIcon />}
+                    </IconButton>
+                  )}
+                  {headerAction}
+                </>
+              }
+              sx={{ pr: 1, width: '100%', ...sxs?.header }}
+            />
+          </AccordionSummary>
+        )}
+        {/* endregion */}
+        {!isMaximized && <AccordionDetails sx={{ p: 0 }}>{renderCardContent()}</AccordionDetails>}
+      </Accordion>
+      {isMaximized && renderCardContent()}
```

Comment:
Card content rendered outside of accordion in maximized mode due to fixed height in accordion preventing the content to fully display